public/index.html

```diff
@@ -52,6 +52,11 @@
 		<div class="container-fluid" style="max-width:90%;">
 			<div class="well row attributesWrapper" style="margin-top: 1%;">
 				<form action="#" class="argumentsForm">
+					<div class = "Title">
+						<img src="images/odu-logo.png"  alt = "ODU Logo" >
```

Comment:
You might want to use https://github.com/oduwsdl/logo/blob/master/glider-trans.png logo instead.